src/approaches/nsrt_learning_approach.py

```diff
@@ -61,6 +63,8 @@ def _learn_nsrts(self, trajectories: List[LowLevelTrajectory],
         save_path = utils.get_approach_save_path_str()
         with open(f"{save_path}_{online_learning_cycle}.NSRTs", "wb") as f:
             pkl.dump(self._nsrts, f)
+        if CFG.compute_sidelining_objective_value:
```

Comment:
hopefully we can also use this to see whether the objective value improves with each iteration of backchaining

Reply:
that's, uhh... going to be annoying...
- if we want to do what you said, we can't put this method in `nsrt_learning_approach.py`, since the strips learners don't have access to the approach object
- we can't put this method in the strips learners since we need to update the metrics dictionary of the `NSRTLearningApproach` with this information so that we can analyze it in the scripts, but the `NSRTLearningApproach` only sees the top-level method `learn_nsrts_from_data()`, not any of the details about the strips learners
- we can't put this method in utils.py because it has dependencies on planner.py, which is circular

the only solution i can think of is to put it in planner.py.... thoughts on that?

Reply:
Is there a version of the second bullet where we could pass the metrics dict into the helper function?

Reply:
no because there are three layers: the Approach, `nsrt_learning_main.py`, and the strips learners. each one sees itself and the thing after it. so Approach can't see the strips learners.

Reply:
aight then I vote to leave as is I guess